test/pmi_client.c

```diff
@@ -334,30 +328,6 @@ static int test_item5(void)
 static int test_item6(void)
 {
     int rc = 0;
-    char val[100];
-    const char *tkey = __func__;
-    const char *tval = __FILE__;
-
-    if (PMI_SUCCESS != (rc = PMI_KVS_Put(jobid, tkey, tval))) {
-        log_fatal("PMI_KVS_Put %d\n", rc);
-        return rc;
-    }
-
-    if (PMI_SUCCESS != (rc = PMI_KVS_Get(jobid, tkey, val, sizeof(val)))) {
```

Comment:
I would like to emphasis that this test works for PMI (version 1). So motivation of the test is verification if PMIx is compatible with PMI (version 1)  or not.
So if PMIx has limitation for this specific capability we can/must describe one in PMIx limitation section. In this case I would like to prefer not remove test that is valid for PMI and just check failure as expected result for PMIx.